Cubical/Foundations/Path.agda

```diff
@@ -17,6 +17,10 @@ cong′ : ∀ {B : Type ℓ'} (f : A → B) {x y : A} (p : x ≡ y)
       → Path B (f x) (f y)
 cong′ f = cong f
 
+PathP→Path : ∀ {ℓ} {A B : Type ℓ} {C : A ≡ B} {a : A} {b : B}
+  → PathP (λ i → C i) a b → transport C a ≡ b
+PathP→Path {C = C} p i = transp (λ j → C (i ∨ j)) i (p i)
```

Comment:
This is kind of the same as: https://github.com/agda/cubical/blob/master/Cubical/Foundations/Prelude.agda#L174

Reply:
Oops, I'll use that one.